packages/SwingSet/src/types-internal.js

```diff
@@ -0,0 +1,17 @@
+import './types.js';
+import './types-exported.js';
+
+export {};
+
+/**
+ * @typedef { string } VatID
+ * @typedef { string } MeterID
+ * @typedef { { meterID?: MeterID } } HasMeterID
+ * @typedef { import('./types-exported.js').DynamicVatOptionsWithoutMeter } DynamicVatOptionsWithoutMeter
+ *
+ * // used by vatKeeper.setSourceAndOptions(source, RecordedVatOptions)
+ *
+ * @typedef { DynamicVatOptionsWithoutMeter & HasMeterID } InternalDynamicVatOptions
+ * @typedef { StaticVatOptions | { InternalDynamicVatOptions & HasMeterID } } RecordedVatOptions
```

Comment:
I'm confused why this doesn't require a `import('./types-exported.js').StaticVatOptions`